### ISSUE TITLE
Can we get solutions to our problem sets?

### ISSUE THREAD

**Issue by ZhengErWei** [UC-MACSS/persp-model_W18#136]
In my feedback from PS2, one point is deducted for calculating wrong vcv. But the vcv results differ from different initial guesses and bounds of parameters we use. I wonder if it's possible to know what's wrong with our code or what the sample solution looks like. Thanks.

**Comment by xiuyuanzhang** [comment]
Yes, I second this suggestion. I also would want to know which vcv values I got wrong and how did I get it wrong. I know it is lots of work for the TAs but I would really benefit more from the assignments if I get more constructive comments. Thank you!

**Comment by Otamio** [comment]
Yes, I guess sample solutions are helpful for us to correct our errors and prepare the exam.

**Comment by rickecon** [comment]
No. But you can come by my office and I can show you the solutions after your assignments have been graded. Further, you can come to my office hours and the TA office hours and lab hours. All of us have the solutions to the problems.
